portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,34 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> list;
+  
+  @Override
+  public void init(){
+      list = new ArrayList<>();
+      list.add("Pineapple on pizza is pretty good");
+      list.add("Dogs are overrated");
+      list.add("Alpacas are awesome");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Budi!</h1>");
+    init();
```

Comment:
Is the call to init() necessary here? I am not sure how Servlets are created, but I would assume init() to be called once at or soon after creation of the Servlet

Reply:
Yea you're right, init() is called automatically on creation.